src/contracts/interfaces/IDelegationManager.sol

```diff
@@ -526,6 +526,10 @@ interface IDelegationManager is ISignatureUtils, IDelegationManagerErrors, IDele
     /// @notice return address of the beaconChainETHStrategy
     function beaconChainETHStrategy() external view returns (IStrategy);
 
+    /// @notice Backwards-compatible interface to return the `MIN_WITHDRAWAL_DELAY_BLOCKS` value
+    /// @dev Previous value in storage was deprecated. See `__deprecated_minWithdrawalDelayBlocks`
+    function minWithdrawalDelayBlocks() external view returns (uint32);
+
```

Comment:
We need to get rid of the all-caps version of the method, or we'll have to support both permanently.

We can't afford the extra bytecode.

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/951/commits/48236927e0bfb33c73d063ad02654ddc532e02ae